brew_list.txt

```diff
@@ -6,3 +6,5 @@ node
 packer
 pkg-config
 wget
+python
```

Comment:
I dont think python is necessary here, as I don't seem to have that in my brew list.